.github/workflows/coveralls.yml

```diff
@@ -2,8 +2,14 @@ name: Coveralls
 
 on: ["push", "pull_request"]
 
+permissions:
+  contents: read
+
 jobs:
   mobx:
+    permissions:
+      checks: write  # for coverallsapp/github-action to create new checks
```

Comment:
@naveensrinivasan could you elaborate a bit more why a write permissions is needed for coveralls? I'd expect only read would be enough, and I didn't see anything in there docs?

Reply:
Let me get back to you on this. It's been a while. Thanks

Reply:
@naveensrinivasan friendly bump

Reply:
Can someone please take over this PR? I don't have the bandwidth now.  Thanks